_layouts/plantillablog.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
```

Comment:
En principio lo que se pedía es hacer una plantilla basada en la que ya había. Así te habrías ahorrado repetir todo el código que ya está en la plantilla original. Quedaría de la siguiente manera:
```suggestion
---
layout: layout
---
<h3>Últimas entradas</h3>
        <ul>
          {% for post in site.posts %}
            <li>
              <a href="{{ site.baseurl }}/{{ post.url }}">{{ post.title }}</a> {{ post.date | date_to_string }}
              <p>{{ post.excerpt }}</p>
            </li>
          {% endfor %}
        </ul>
```

Reply:
Cierto! Muchas gracias! No había leído bien el enunciado.

Lo he modificado y lo he integrado en mi proyecto, cambiándolo en local y después con un `git push`. ¿Si el PR hubiera permanecido abierto, el botón de _Apply suggestion_ me hubiera realizado el cambio en mi repositorio?

Muchas gracias!

Reply:
Pues buena pregunta... Vamos a hacer la prueba! Abro la PR de nuevo y así lo puedes probar!

Reply:
Buenas, ¿te actualizó el cambio al aceptar la sugerencia? ¿O tuviste que hacerlo a mano?

Reply:
No lo había podido probar todavía, el cambio lo realicé a mano... Pero lo acabo de probar.

Y sí, aplica el cambio sugerido en el comentario. Solo que como en el comentario solo eliminabas la primera línea añadiendo en su lugar el código nuevo y manteniendo todo el resto, es precisamente eso lo que ha hecho.


👍